Source/JavaScriptCore/API/JSManagedValue.mm

```diff
@@ -85,7 +85,7 @@ - (instancetype)initWithValue:(JSValue *)value
     if (!value)
         return self;
 
-    JSC::JSGlobalObject* globalObject = toJS([value.context JSGlobalContextRef]);
+    JSC::JSGlobalObject *globalObject = toJS((value.context).JSGlobalContextRef);
```

Comment:
Isn't JSGlobalObject* still a C++ type? So the `*` could stay where it was?
Also it seems the parenthesis may not be needed on the right side. Yielding `value.context.JSGlobalContextRef`.